library.json

```diff
@@ -26,7 +26,9 @@
       "examples/*"
     ]
   },
-  "frameworks": ["arduino", "energia", "wiringpi"],
-  "platforms": "*",
+  "frameworks": "arduino",
+  "platforms": [
+    "espressif32"
```

Comment:
Fix this fork compatibility since other board compatibility is not ensured in new implementations